Cargo.toml

```
@@ -8,3 +8,4 @@ edition = "2021"
 [dependencies]
 termion = "1"
 unicode-segmentation = "1"
+clipboard = "0.5"
```

Please keep the dependencies in lexicographical order.
```suggestion
[dependencies]
clipboard = "0.5"
termion = "1"
unicode-segmentation = "1"
```